CWE/events/italy.txt

```diff
@@ -54,86 +95,6 @@ country_event = {
 	}
 }
 
-#ICP foundation
```

Comment:
Please do not remove these events unless there is an immediate replacement for them.

Reply:
I will be adding a first stage of that later today and if I can't today by the end of the week, the Foundation event is supposed to happen before gamestart and the rest of the chain doesn't really fit what will be added nor will work with it

Reply:
Alright